uf_auth/README.md

```diff
@@ -4,9 +4,11 @@
 
 Execute the following instructions in the directory of this folder
 
-`brew install chromedriver `
+`brew tap homebrew/cask`
```

Comment:
I'm not sure if adding cask is necessary, but it looks like an improvement none the less. Thanks for contributing!